### DISCUSSION TITLE
use custom icon for workspace (and otherwise make it stand out visually in the left bar)

### DISCUSSION THREAD

**Discussion by m10d** [flxzt/rnote#825] [Q&A]
I have, for example, several chemistry related workspaces.
The erlenmeyer flask is great ... for if I have just a single.  But multiple, it does not stand out (and there is nothing for me to remember colors by)

Is there any way to:

- add new icons to the set that come up in the edit workspace
- change font of the text (only the first two characters are displayed - I'd need at least ~4 for that to mean anything in my use case

Thank you!

**Comment by flxzt** [comment]
I think there is already a feature request for custom icons. Customizing the workspace label further would also be nice, I agree.

**Comment by m10d** [comment]
closing for https://github.com/flxzt/rnote/issues/598
